cmd/site-replication.go

```diff
@@ -72,6 +72,10 @@ var (
 		Cause: errors.New("peer not found"),
 		Code:  ErrSiteReplicationInvalidRequest,
 	}
+	errSRRequestorNotFound = SRError{
+		Cause: errors.New("requesting site not found in site replication config"),
```

Comment:
Q: Can we state which site which was not found? Or is this intended to say that the site specified as the initial alias is not in the replication config (e.g. for mc admin replicate add/remove)

Reply:
the requesting site here is the initial alias in `mc admin replicate remove`.